test/unit/ReservoirPair.t.sol

```diff
@@ -119,8 +119,22 @@ contract ReservoirPairTest is BaseTest {
         _pair.sync();
 
         // assert
-        (, , , uint16 lIndex) = _pair.getReserves();
+        (,,, uint16 lIndex) = _pair.getReserves();
         Observation memory lObs = _oracleCaller.observation(_pair, lIndex);
-        assertEq(lObs.logAccLiquidity, 470050);
+        assertEq(lObs.logAccLiquidity, 470_050);
+    }
+
+    function testCheckedTransfer_RevertWhenTransferFail() external allPairs {
```

Comment:
Should test two types of fails (does mockCall let you trigger a revert?):

- Return false
- Revert

Reply:
unfortunately the current forge-std cheatcodes do not include mocking a revert (although this is something I've found myself and others desiring multiple times). 

So if we can't mock a call, the next best thing is to manually set the wallet balance to 0 and make it revert. Added the test case